### PR TITLE
Fixes the readme

### PR DESCRIPTION
**Pull request by jwdebelius**

### CODE REVIEW

File: bloom/filter_seqs_from_biom.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def remove_seqs(table, seqs):
     biom.Table
     """
     filter_seqs = {str(s) for s in seqs}
-    _filter = lambda v, i, m: i not in filter_seqs
+    _filter = lambda v, i, m: i in filter_seqs
```
Comment (wasade) [inline bloom/filter_seqs_from_biom.py]: isn't this the opposite of what we want?

File: bloom/filter_seqs_from_biom.py [hunk 2]
```diff
@@ -45,8 +45,7 @@ def remove_seqs(table, seqs):
     """
     filter_seqs = {str(s) for s in seqs}
     _filter = lambda v, i, m: i in filter_seqs
-    table.filter(_filter, axis='observation', invert=True)
-    return table
+    return table.filter(_filter, axis='observation', inplace=False)
```
Comment (wasade) [inline bloom/filter_seqs_from_biom.py:48]: i believe this should be inverted given the description of the filter method

File: ipynb/bloom_example.ipynb [hunk 3]
```diff
@@ -66,7 +66,15 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gunzip: can't stat: ../data/30_seqs.fna.gz (../data/30_seqs.fna.gz.gz): No such file or directory\r\n"
```
Comment (wasade) [inline ipynb/bloom_example.ipynb]: if the notebook contains output, it should not be erroneous output

File: README.md [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
+## Installion
+
+Download the repository from [GitHub](https://github.com/knightlab-analyses/bloom-analyses), and navigate into that directory. The repository can be installed by running the following commands in the linux terminal.
 
-To reproduce the analyses in the analyses in `ipynb`, run the following commands
 ```
-conda create -n bloom pip python=3 numpy scikit-bio jupyter seaborn matplotlib=1.5.1
+conda create -n bloom pip python=3 numpy jupyter seaborn matplotlib=1.5.1 statsmodels
+source activate bloom
+pip install scikit-bio==0.5.0
```
Comment (wasade) [inline README.md]: why the 0.5.0 requirement?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
 pip install git+https://github.com/amnona/heatsequer.git
```
Comment (wasade) [inline README.md]: can this line involving heatsequer be removed please?

File: README.md [hunk 6]
```diff
@@ -1,12 +1,22 @@
...
+pip install emperor --pre
 pip install -e .
-```
\ No newline at end of file
+```
+## Notebooks
```
Comment (wasade) [inline README.md:16]: we explicitly mention in the submitted manuscript that there is a notebook which shows how to apply the filter.

Comment (wasade) [reply README.md:16]: ...can the current notebook be added to the descriptions below?

File: bloom/filter_seqs_from_biom.py (hunk 2 above)
Comment (mortonjt) [reply bloom/filter_seqs_from_biom.py:48]: that is correct

**Comment by mortonjt** [comment]
Here is is what needs to be done to fix the failing test
1. `bloom/filter_seqs_from_biom.py` line 40: add `invert=True`
2. change `seqs_file = '30_seqs.fna'` to `seqs_file = 'newbloom.all.fna'`
3. `git rm 30_seqs.fna.gz`
4. Remove the gunzip command

That should be it.  I'll merge this once those changes are in.

**Review by wasade — COMMENTED** [review]
:+1:

Just to verify, this has been run on say the ag data (in repo) or other and sanity checked?

**Comment by jwdebelius** [comment]
@mortonjt, @wasade: Should be good to go.

Revert is suprisingly awesome.

**Comment by mortonjt** [comment]
Wooooo!  Thanks @jwdebelius !

**Pull request merged**